### ISSUE TITLE
How do I contribute to the project? Can you describe the steps to build?

### ISSUE THREAD

**Issue by wevertoonsilva** [WINOFFRG/limeplay#37]

**Comment by WINOFFRG** [comment]
Hey @wevertoonsilva! First you need to have [PNPM](https://pnpm.io/installation#using-npm) tool, It is quite simple to install. Once installed just run `pnpm install` in root directory. I still have to setup Storybook for independent component/hook development. But currently I am managing it via the [Home Site](https://limeplay.me/player) Just open the `/player` which will just load the basic player with most hooks in place. Also, `main` is currently outdated and `migration` is the latest branch and I have a lot of changes on local which I plan to complete by this week. 

Kindly raise an issue first for the respective contribution, I'll help you go ahead. Thanks!

**Comment by wevertoonsilva** [comment]
Thank you @WINOFFRG, everything worked out here. I studied the project and I'm integrating it into my website, with that I found some inconsistencies that were simple to resolve, but I believe you've already resolved most of them in the migration branch.

I'd be happy if you updated the migration branch so I could download it, even if it's not ready yet.

Congratulations, it's a very promising project.
